consensus/consortium/v2/consortium.go

```diff
@@ -603,10 +598,14 @@ func (c *Consortium) processSystemTransactions(chain consensus.ChainHeaderReader
 		}
 		spoiledVal := snap.supposeValidator()
 		signedRecently := false
-		for _, recent := range snap.Recents {
-			if recent == spoiledVal {
-				signedRecently = true
-				break
+		if c.chainConfig.IsBuba(header.Number) {
```

Comment:
This is just a placeholder for next hardfork, will change to IsOlek when https://github.com/axieinfinity/ronin/pull/252 is merged